docker/.env

```diff
@@ -29,3 +29,5 @@ LIGHTDASH_CONFIG_FILE=/usr/app/lightdash.yml
 AUTH_GOOGLE_OAUTH2_CLIENT_ID=
 AUTH_GOOGLE_OAUTH2_CLIENT_SECRET=
 AUTH_DISABLE_PASSWORD_AUTHENTICATION=false
+
+ALLOW_MULTIPLE_ORGS=false
```

Comment:
I think we should have this enabled on dev mode. Is going to be super useful :) 
And perhaps for PR envs as well so we can easily test it. I can't find reasons why not too. 🤔